dChatServer/PlayerContainer.cpp

```diff
@@ -13,6 +13,23 @@
 #include "dConfig.h"
 #include "MessageType/Chat.h"
 
+using json = nlohmann::json;
+
+const json PlayerData::to_json() const {
+	json data;
+	data["id"] = this->playerID;
+	data["name"] = this->playerName;
+	data["gm_level"] = this->gmLevel;
+	data["muted"] = this->GetIsMuted();
+
+	json zoneID;
+	zoneID["map_id"] = std::to_string(this->zoneID.GetMapID());
+	zoneID["instance_id"] = std::to_string(this->zoneID.GetInstanceID());
+	zoneID["clone_id"] = std::to_string(this->zoneID.GetCloneID());
+	data["zone_id"] = zoneID;
```

Comment:
```suggestion
	json& zoneID = data["zone_id"];
	zoneID["map_id"] = std::to_string(this->zoneID.GetMapID());
	zoneID["instance_id"] = std::to_string(this->zoneID.GetInstanceID());
	zoneID["clone_id"] = std::to_string(this->zoneID.GetCloneID());
```
I think this is a better format for json personally and should result in 1 less copy (though compilers may see this and do it for you)